src/System.Management.Automation/commands/utility/FormatAndOutput/out-console/OutConsole.cs

```diff
@@ -102,6 +98,7 @@ protected override void ProcessRecord()
         {
             if (Transcript)
             {
+                Host.UI.TranscribeOnly = true;
```

Comment:
> as the intent of -Transcript (as far as I can tell) is to **suppress output to the pipeline**

Sadly it is exact opposite of how `Out-Default -Transcript` is implemented. By default `Out-Default` does not output anything, but display input on host. With `-Transcript` it pass thru the input, but making following `Out-Default`/`Out-Host` to not display their input:
```powershell
1..10 | Out-Default -Transcript | Out-Host
```
But by replacing `Out-Host` with `Write-Host` you can see, that output is really here:
```powershell
1..10 | Out-Default -Transcript | Write-Host
```